democracy_club/templates/data_apis/voting_information_api.html

```diff
@@ -0,0 +1,97 @@
+{% extends "data_apis/base.html" %}
+{% load markdown_filter %}
+
+{% block main_content %}
+
+    <h2>Voting Information APIs</h2>
+
+    <p>We collect and standardise election data including seats,
+        polling stations, candidates and more; everything you need
+        to get your users informed and voting.
+        This information is endorsed and used by the Electoral
+        Commission and powers many other organisations.</p>
+
+    <ul class="ds-grid" style="--gridCellMin: 20ch; --gridGap: 2rem;">
+        <li class="ds-card">
+            <div class="ds-card-body">
+                <h5>Hobbyists</h5>
```

Comment:
This heading is just below a `h2`, so can you make these `h3` elements? If you need to control the size, you can e.g `<h3 class="ds-h5">`